hooks.go

```diff
@@ -21,14 +21,38 @@ func (hooks LevelHooks) Add(hook Hook) {
 	}
 }
 
+// multiErr may contain multiple errors.
+type multiErr []error
+
+func (e multiErr) Error() string {
+	if e == nil || len(e) == 0 {
+		return ""
+	}
+	return e[0].Error()
```

Comment:
I would expect something more along the lines of: https://play.golang.org/p/P6EyQ4prr3s

It's debatable if adding the \n is good though.

Reply:
That's a good point. I really wish there were a standardize way to handle multiple errors in Go.

This makes me think it may actually be a good idea, even now, to export the `multiErr` type, so that consumers of the library can get at the underlying error values, if they want to.